src/com/aca/exceptions/WrongProductData.java

```diff
@@ -0,0 +1,6 @@
+package com.aca.exceptions;
+
+import java.io.IOException;
+
+public class WrongProductData extends IOException {
```

Comment:
put here functionality to provide some message About causing, for instance provide the path where we don't found any product